apis/apiloggi.py

```diff
@@ -0,0 +1,48 @@
+from rastreio import db
+
+import configparser
+import requests
+
+config = configparser.ConfigParser()
+config.read("bot.conf")
+
+api_key = config["LOGGI"]["api_key"]
+email = config["LOGGI"]["email"]
+final_codes = {int(x) for x in config["LOGGI"]["final_codes"].split(", ")}
```

Comment:
```suggestion
final_codes = {int(x) for x in config["LOGGI"]["final_codes"].split(",").strip()}
```
Para evitar que a config quebre por falta de espaço.

Reply:
ah, boa!